Remove TaskInstance.log_filepath attribute (#15217)
mistercrunch/Airflow: UPDATING.md

-->

### Remove `TaskInstance.log_filepath` attribute

This method returned incorrect values for a long time, because it did not take into account the different

> logger configuration and try retires. We're also starting to support more advanced tools that don't use

Reviewer: ```suggestion
logger configuration and task retries. We have also started supporting more advanced tools that don't use
```
